draft-ietf-tls-tls13.md

```diff
@@ -3908,11 +3929,13 @@ the TLS PRF. This document replaces the PRF with HKDF, thus requiring
 a new construction. The exporter interface remains the same. If context is
 provided, the value is computed as:
 
-    HKDF-Expand-Label(exporter_secret, label, context_value, key_length)
+    HKDF-Expand-Label(Secret, label, context_value, key_length)
+
+Where Secret is either the early_exporter_secret or the exporter_secret.
```

Comment:
Can this be expanded to say when early_exporter_secret should be used vs exporter_secret? E.g. something like exporter_secret must be used for providing the RFC 5705 interface?